scripts/damage-log.js

```diff
@@ -110,6 +110,13 @@ class DamageLog {
 			},
 		},
 		dnd5e: DamageLog.#DND_ATTRIBUTES,
+		dragonbane: {
+			hp: {
+				value: "hitPoints.value",
+				min: "0",
```

Comment:
The `min` parameter is not required.  It will default to zero if not provided.  I'll remove this myself after I've merged your change.